guides/common/modules/proc_obfuscating-hostnames-and-ip-addresses.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="obfuscating-hostnames-and-ip-addresses"]
+= Obfuscating hostnames and IP addresses
+
+You can use {ProjectName} to obfuscate hostnames and IP addresses when uploading host data to Insights. 
+Obfuscation ensures that sensitive identifying information is not exposed. 
+When obfuscation settings are enabled, {Project} obfuscates hostnames and IP addresses. 
+If obfuscation settings are enabled and the host has its own Insights client obfuscation settings, {Project} applies the client settings. 
+If obfuscation settings are disabled, {Project} does not obfuscate hostnames or IP addresses, regardless of client settings.
+
+By default, {Project} applies global obfuscation settings unless an Insights client on the host overrides the settings with its own configuration.
+This procedure describes how {Project} applies obfuscation settings.
+
+.Prerequisites
+* Your hosts have Insights clients installed and configured.
+
+.Procedure
+. Navigate to *Insights* > *Inventory Upload*.
+. Enable the *Obfuscate host names* and *Obfuscate host ipv4 addresses* settings to obfuscate host names and ipv4 addresses sent to the {RHCloud}.
+
+.Verifying obfuscation
```

Comment:
For this I would have them download the report in *Insights* > *Inventory Upload* -> pick the org dropdown selector and then click download report and have them view that. Satellite will always show the real hostname in the UI because that is what subscription-manager uses